<launch>

<!-- Launches my listenerMotorAuto.cpp file to initialize a base controller 
  that move_base communicates with to send motor inputs. This links autonomy to motors 
<node pkg="gen2bot" type="mine" name="motorRuns" output="screen">

    <param name="wheel_multiplier" value="1.1" type="double"/>

    <rosparam file="$(find gen2bot)/config/PID.yaml" command="load"/> 
</node> -->

<node pkg="gen2bot" type="listenerMotorAuto" name="wheels" output="screen">
    <param name="initialOutput" value="0.2"/>
    <param name="isInitialState" value="true"/>
</node>

</launch>
